docs/docs/testing/semantic-dom-diff.md

```diff
@@ -26,10 +26,10 @@ While `semantic-dom-diff` can be used standalone (see below), it most commonly u
 > If you are using `@open-wc/testing` this is already done for you.
 
 ```javascript
-import 'chai/chai.js';
+import * as chai from 'chai';
 import { chaiDomDiff } from '@open-wc/semantic-dom-diff';
 
-window.chai.use(chaiDomDiff);
+chai.use(chaiDomDiff);
```

Comment:
This needs to be `const chai = use(chaiDomDiff);` actually.

https://github.com/chaijs/chaijs.github.io/pull/206#pullrequestreview-2038899153